packages/backend/server/tests/copilot.spec.ts

```diff
@@ -533,3 +540,25 @@ test('should be able to register test provider', async t => {
   await assertProvider(CopilotCapability.ImageToImage);
   await assertProvider(CopilotCapability.ImageToText);
 });
+
+test.skip('should be able to run workflow', async t => {
```

Comment:
I didn't see this testcase gets enabled in upcoming prs

Reply:
correct test added to #7159 now, and i update a comment here